polaris-migrator/package.json

```diff
@@ -33,6 +33,7 @@
     "generate": "plop"
   },
   "dependencies": {
+    "@babel/runtime": "^7.20.13",
```

Comment:
I don't think these babel dependencies are needed. @samrose3 and I were trying to figure out resolving this issue when running the migrations across `shopify/polaris`. It seems to happen on `postcss` migrations only.

```
ReferenceError: regeneratorRuntime is not defined
    at /Users/alexpage/Sites/shopify/polaris/polaris-migrator/node_modules/postcss/lib/lazy-result.js:387:4
    at /Users/alexpage/Sites/shopify/polaris/polaris-migrator/node_modules/postcss/lib/lazy-result.js:444:4
    at Object.<anonymous> (/Users/alexpage/Sites/shopify/polaris/polaris-migrator/node_modules/postcss/lib/lazy-result.js:539:4)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Module._compile (/Users/alexpage/Sites/shopify/polaris/node_modules/pirates/lib/index.js:136:24)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Object.newLoader [as .js] (/Users/alexpage/Sites/shopify/polaris/node_modules/pirates/lib/index.js:141:7)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
/Users/alexpage/Sites/shopify/polaris/polaris-migrator/node_modules/postcss/lib/lazy-result.js:391
      var _runAsync = _asyncToGenerator( /*#__PURE__*/regeneratorRuntime.mark(function _callee() {
```

Reply:
I was able to get around this error by running the migration from the `polaris-react` directory instead of the root. From my investigation, it appears it is related to how `jscodeshift` is performing babel transforms on files, though it shouldn't be doing so on `node_modules`. However, here it is trying to on postcss.